client/src/components/Summary.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Summary = props => (
+  <div id="summary" style={props.border}><h1>Summary</h1>
```

Comment:
Is there are reason that you're using headings instead of setting the font-size?